tests/modules/encounters/test_models.py

```diff
@@ -1,11 +1,15 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=invalid-name,missing-docstring
 
+from app.modules.users.models import User
```

Comment:
These were moved inside tests as outside they were causing some nasty circular dependencies IIRC, JP would know the details as he did the change.